api/v1alpha1/gatling_types.go

```diff
@@ -114,6 +114,10 @@ type TestScenarioSpec struct {
 	// +kubebuilder:validation:Optional
 	Parallelism int32 `json:"parallelism,omitempty"`
 
+	// (Optional) Gatling simulation format, supports `bundle` and `gradle`. Defaults to `bundle`
+	// +kubebuilder:validation:Optional
+	SimulationsFormat string `json:"simulationsFormat,omitempty"`
```

Comment:
I want to validate to accept only `bundle` or `gradle`.
Like `+kubebuilder:validation:Enum=bundle;gradle`

Reply:
good one, applied 👍